src/block-management/block-holder.js

```diff
@@ -238,6 +239,7 @@ export default compose( [
 			isLastBlock,
 			isSelected,
 			name,
+			clientId,
```

Comment:
`withSelect` only add new props to the component, so it should not be necessary to explicitly add `clientId` which is already a prop given from the parent component here.